packages/hint/tests/lib/engine.ts

```diff
@@ -874,3 +874,46 @@ test.serial('executeOn should return all messages', async (t) => {
 
     t.is(result.length, 2);
 });
+
+test.serial('executeOn should forward content if provided', async (t) => {
```

Comment:
We started using `'` around coding related things such as `'executeOn'`, so let's use that in the future. 

I'll make this change before merging